.github/workflows/test.yml

```diff
@@ -0,0 +1,132 @@
+name: Timber Tests
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    types: [opened, synchronize, ready_for_review]
+
+jobs:
+  phpunit:
+    runs-on: ubuntu-latest
+
+    services:
+      mysql:
+        image: mariadb:latest
+        env:
+          MYSQL_ALLOW_EMPTY_PASSWORD: true
+          MYSQL_ROOT_PASSWORD:
+          MYSQL_DATABASE: wordpress_test
+        ports:
+          - 3306
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+
+    continue-on-error: ${{ matrix.experimental == true }}
+    strategy:
+
+      fail-fast: false
+      matrix:
+        php: ['7.4', '7.3', '7.2'] # , '7.1', '7.0'
```

Comment:
Per https://github.com/timber/timber/issues/1863#issuecomment-760493487, I actually don't think we need to worry about anything other than 7.4 right now. (We can support 8.0 in a future version.)

Reply:
We should run tests on the lowest supported PHP version.

Reply:
Good news! Done.

Reply:
```json
  "require": {
    "php": "^7.0",
```

Please at least add 1 test on low PHP version.

Reply:
Support for <= 7.3 will be dropped per discussion linked above. The current change is correct.

Reply:
# 😲

Reply:
@szepeviktor Tests actually don't pass on PHP 7.0, neither on 7.1. Maybe should I update `composer.json`?

Reply:
There's also a serious issue about Twig minimum version that's not tested at all (and don't pass).

Reply:
No, it is enough to run on PHP 7.4

Reply:
@nlemoine that Twig thing is only an issue on older PHP versions though, right? FWIW, we will require Twig >= 2.10. See: https://github.com/timber/timber/issues/2407

Reply:
[Not at all](https://github.com/nlemoine/timber/runs/2317169569?check_suite_focus=true). Please consider requiring a more recent version of Twig (at least `^2.4 || ^3.0`). 

From #2436 : 

`twig/cache-extension` has a `twig:^2.4` constraint and it would be a shame to release the brand new Timber 2.0 with an extension that's already not maintained anymore.

Reply:
@nlemoine I see, thanks. I agree, and I think fixing #2407 will address your concern. The plan is to replace that extension with twig/cache-extra.